qutebrowser/config/configtypes.py

```diff
@@ -1151,6 +1162,17 @@ class UserStyleSheet(File):
     def __init__(self):
         super().__init__(none_ok=True)
 
+    def transform(self, value):
+        if not value:
+            return None
+        path = super().transform(value)
+        path = os.path.expandvars(path)
```

Comment:
Before `os.path.expanduser` was also called which isn't anymore - but actually both `expanduser` and `expandvars` should be called in `File.transform` IMHO. I'm not sure why `expandvars` isn't there.